docs/toc.yml

```diff
@@ -566,13 +238,379 @@
+          href: reference/manifest/action.md
+        - name: AllFormFactors
+          href: reference/manifest/allformfactors.md
+        - name: Control
+          href: reference/manifest/control.md
+        - name: CustomTab
+          href: reference/manifest/customtab.md
+        - name: Event
+          href: reference/manifest/event.md
+        - name: ExtensionPoint
+          href: reference/manifest/extensionpoint.md
+        - name: DesktopFormFactor
+          href: reference/manifest/desktopformfactor.md
+        - name: FunctionFile
+          href: reference/manifest/functionfile.md
+        - name: GetStarted
+          href: reference/manifest/getstarted.md
+        - name: Group
+          href: reference/manifest/group.md
+        - name: Hosts
+          href: reference/manifest/hosts.md
+        - name: Icon
+          href: reference/manifest/icon.md
+        - name: MobileFormFactor
+          href: reference/manifest/mobileformfactor.md
+        - name: OfficeMenu
+          href: reference/manifest/officemenu.md
+        - name: OfficeTab
+          href: reference/manifest/officetab.md
+        - name: Page
+          href: reference/manifest/page.md
+        - name: Resources
+          href: reference/manifest/resources.md
+        - name: Scopes
+          href: reference/manifest/scopes.md
+        - name: Script
+          href: reference/manifest/script.md
+        - name: SourceLocation (custom functions)
+          href: reference/manifest/customfunctionssourcelocation.md
+        - name: Supertip
+          href: reference/manifest/supertip.md
+        - name: SupportsSharedFolders
+          href: reference/manifest/supportssharedfolders.md
+        - name: WebApplicationInfo
+          href: reference/manifest/webapplicationinfo.md
+    - name: API open specifications
+      href: reference/openspec/openspec.md
+    - name: Error codes
+      href: reference/javascript-api-for-office-error-codes.md
+  - name: Resources
+    href: resources/resources-links-help.md
+    displayName: Excel, Word, OneNote, Project, PowerPoint, Visio, Outlook
+
+- name: Excel add-ins
+  items: 
+  - name: Excel add-ins documentation
+    href: excel/index.md
+    displayName: Excel
+  - name: Overview
+    href: excel/excel-add-ins-overview.md
+    displayName: Excel
+  - name: Build your first Excel add-in
+    items: 
+    - name: Task pane
+      href: quickstarts/excel-quickstart-jquery.md
+      displayName: Excel
+    - name: Task pane (Angular)
+      href: quickstarts/excel-quickstart-angular.md
+      displayName: Excel
+    - name: Task pane (React)
+      href: quickstarts/excel-quickstart-react.md
+      displayName: Excel
+    - name: Task pane (Vue)
+      href: quickstarts/excel-quickstart-vue.md
+      displayName: Excel
+  - name: Excel JavaScript API
+    items:
+    - name: Overview
+      href: reference/overview/excel-add-ins-reference-overview.md
+      displayName: Excel
+    - name: Excel add-in tutorial
```

Comment:
Can we move the Excel Tutorial up a level? All the other tutorials are at the same level as the quick starts.

Reply:
Valid suggestion; I debated where to put the "Excel add-in tutorial" node. The Excel section is a bit different than the other host sections, because Excel actually has 2 flavors of quick starts and tutorials:  one flavor involves only the Excel JS APIs, while the other flavor is specific to Custom Functions. 

That said, I think I agree with you here -- structure should be consistent across the various host sections. I'll move this node as you've suggested.